src/main/kotlin/Car.kt

```diff
@@ -0,0 +1,22 @@
+import java.lang.IllegalArgumentException
+
+class Car {
+    private var position: Int = 0
```

Comment:
프로퍼티의 getter 함수를 노출하면서 setter 함수에 대한 접근을 제한하려면 setter 함수를 private으로 지정하면 됩니다.

```suggestion
    var position: Int = 0
        private set
```

Reply:
[5e30ace](https://github.com/next-step/kotlin-racingcar/pull/80/commits/5e30ace3f242a6e66a7bc8f8585d25c17ff61717) 수정 반영 하였습니다.
private set 선언하여 getter 함수를 굳이 선언하지 않아도 되어 좋은거 같습니다.
혹시 생성자에 선언된 name 은 별도로 private set 선언을 할 수 없을까요?